backend/typescript/models/entity.mgmodel.ts

```diff
@@ -0,0 +1,35 @@
+import mongoose, { Schema, Document, model } from "mongoose";
+
+export interface IEntity extends Document {
```

Comment:
Renamed to Entity, there is an _id field that is automatically created by Mongo. What would be the benefits of having 2?

Reply:
Just to assert that it's not nullable. I think we should also just use `id` (string representation of object id) instead of `_id` (actual object id)

Without adding `id` to the interface:
<img width="800" alt="without id" src="https://user-images.githubusercontent.com/30205929/111419423-d546e200-86bf-11eb-97e9-a129d5a0f364.png">

With `id`:
<img width="800" alt="with id" src="https://user-images.githubusercontent.com/30205929/111419550-0c1cf800-86c0-11eb-9949-ca9039b8648a.png">